test/GridapTests/BiharmonicTests.jl

```diff
@@ -0,0 +1,54 @@
+module BiharmonicTests
```

Comment:
Is this file being run by the test suite?

It seems that it needs to be included into `test/GridapTests/runtests.jl`

Reply:
Done in 2b7c17090a5681cbb34477681087c0f8b0bab04c